### DISCUSSION TITLE
The dark mode can't be shown

### DISCUSSION THREAD

**Discussion by mendax1234** [MrDogeBro/sphinx_rtd_dark_mode#32] [Support] [answered]
l have followed the step to enable the darkm mode, but it just can't be shown.

**Comment by mendax1234** [comment]
> l have followed the step to enable the dark mode, but it just can't be shown.

**Comment by mendax1234** [comment]
![image](https://user-images.githubusercontent.com/62934294/176079289-37f4e921-b0be-4c05-9373-0c51bcadd12a.png)
It just said there is no module but l have installed it. That's strange. Could you help me with it?@MrDogeBro

**Comment by MrDogeBro** [comment, marked as answer]
I don't have enough info to be able to help you. If you want help, please provide more info. Also, this is not an issue with the module itself therefore I am moving this into a discussion. 

To me, at least from the outside, it appears to be an issue on your end where something is not setup properly or something. Please first start by looking up “no module named” issue and see if you can try and fix the issue. 

Maybe you have two versions of python installed and the pip installed to the one that’s not building the docs, I don’t know. But currently there is nothing I can do to help you. So please do some research and see if you can try and find the culprit and fix that. Also please give more information about your setup, what you’ve done, etc. so I can try and give you some pointers as to what may be wrong.

Thanks.

**Reply by mendax1234** [reply]
> Maybe you have two versions of python installed and the pip installed to the one that’s not building the docs, I don’t know. But currently there is nothing I can do to help you. So please do some research and see if you can try and find the culprit and fix that. Also please give more information about your setup, what you’ve done, etc. so I can try and give you some pointers as to what may be wrong.

Yes , you are right! After l try to install it with `pip install sphinx-rtd-dark-mode`. It works well , but l also wonder whether this extension can change the theme color according to the system?
@MrDogeBro

**Reply by MrDogeBro** [reply]
> but l also wonder whether this extension can change the theme color according to the system?

Not currently, but that could be something that could be added in the future.
